### ISSUE TITLE
Thoughts about generated/real images ratio

### ISSUE THREAD

**Issue by thomasryck** [clovaai/deep-text-recognition-benchmark#30]
I am training your model on custom data and I am using transfer learning for the feature extraction part with Resnet-152.

To train the model I use synthetic data since what I will recognize has a limited range and I use also real data. I would like to know what is a good ratio synthetic data/real data and when I can consider that my model as seen enough of synthetic data to train on the real one. 

Best

**Comment by ku21fan** [comment]
Hello, 

It depends on your real test dataset, thus I can not sure what is a good ratio and when is good to train on the real dataset.
However, you can conjecture the good ratio by investigating your real test dataset. 
If your real test dataset seems close to the synthetic dataset, increase the ratio of the synthetic dataset, 
if not, increase the ratio of the real training dataset.

In my case, I usually train the model with synthetic dataset 50% and real dataset 50% by using `--select_data` and `--batch_ratio`, and then evaluate the accuracy, 
After that modulate hyperparameters.

Best
